src/main/java/rs/hybridit/model/BookInstance.java

```diff
@@ -0,0 +1,72 @@
+package rs.hybridit.model;
+import java.util.Date;
+import javax.persistence.*;
+
+@Entity
+@Table(catalog = "dbhybridlibrary", name = "book_instances")
+public class BookInstance {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    @Column(name = "rent_start")
+    private Date rentStart;
```

Comment:
you should use java time api from java 8

https://www.baeldung.com/java-8-date-time-intro